sdk/core/src/policies/transport.rs

```diff
@@ -1,28 +1,20 @@
-#[allow(unused_imports)]
 use crate::policies::{Policy, PolicyResult};
 #[allow(unused_imports)]
+use crate::TransportOptions;
+#[allow(unused_imports)]
 use crate::{Context, HttpClient, Request, Response};
 use std::sync::Arc;
 
-#[derive(Debug, Clone)]
-pub struct TransportOptions {
-    http_client: Arc<dyn HttpClient>,
-}
-
-impl TransportOptions {
-    pub fn new(http_client: Arc<dyn HttpClient>) -> Self {
-        Self { http_client }
-    }
-}
-
 #[derive(Debug, Clone)]
 pub struct TransportPolicy {
-    options: TransportOptions,
+    http_client: Arc<dyn HttpClient>,
 }
 
 impl TransportPolicy {
-    pub fn new(options: TransportOptions) -> Self {
-        Self { options }
+    pub fn new(options: &TransportOptions) -> Self {
```

Comment:
1. I did so in TelemetryPolicy because copying or taking ownership of the options seems pointless for read-only operations.
2. Arc doesn't implement the `Copy` trait, as rustc was complaining hence the explicit `clone` call as well. Taking this by-ref not only saves the copy but resolve that issue as well.